Firestore/Source/Public/FirebaseFirestore/FIRFirestore.h

```diff
@@ -43,22 +43,22 @@ NS_SWIFT_NAME(Firestore)
 - (instancetype)init __attribute__((unavailable("Use a static constructor method.")));
 
 /**
- * Creates, caches, and returns a `Firestore` using the default `FirebaseApp`. Each subsequent
+ * Creates, caches, and returns default `Firestore` using the default `FirebaseApp`. Each subsequent
  * invocation returns the same `Firestore` object.
  *
- * @return The `Firestore` instance.
+ * @return The default `Firestore` instance.
  */
 + (instancetype)firestore NS_SWIFT_NAME(firestore());
 
 /**
- * Creates, caches, and returns a `Firestore` object for the specified _app_. Each subsequent
+ * Creates, caches, and returns default `Firestore` object for the specified _app_. Each subsequent
```

Comment:
_app_ -> app

Reply:
I think the underscores are required to document linking. It follows the pattern from elsewhere.